features/admins/admin_email_blacklist.feature

```diff
@@ -2,35 +2,35 @@
 Feature: Admin email blacklist
   In order to prevent the use of certain email addresses in guest comments
   As an an admin
-  I want to be able to manage a blacklist of email addresses
+  I want to be able to manage a list of banned email addresses
 
-Scenario: Add email address to blacklist
-  Given I am logged in as an admin
-  Then I should see "Blacklist"
-  When I follow "Blacklist"
-  Then I should see "Find blacklisted emails"
-    And I should see "Add email address to blacklist"
+Scenario: Ban email address
+  Given I am logged in as a "policy_and_abuse" admin
+  Then I should see "Banned Emails"
+  When I follow "Banned Emails"
+  Then I should see "Find banned emails"
+    And I should see "Ban email address"
     And I should see "Email"
```

Comment:
Since you're changing the `I have banned the address {string}` step below to fill in the "Email to ban" field instead of just "Email," would you mind changing it here as well (and on the line below)? It's not necessary if you're too busy to fix it, but since both of the forms on that page have the word "Email" in them, I think it'd be nice to keep it unambiguous.